cogs5e/initTracker.py

```diff
@@ -535,6 +535,13 @@ def wrapper(func):
 
             return wrapper
 
+        def mod_or_set(opt_name, old_value):
+            new_value = args.last(opt_name, type_=int)
+            if args.last(opt_name).startswith(('-', '+')):
+                new_value = old_value + args.last(opt_name, type_=int)
```

Comment:
```suggestion
                new_value = old_value + new_value
```
Just reduces the number of calls.